docs/error-messages/compiler-warnings/c4834.md

```diff
@@ -13,7 +13,7 @@ helpviewer_keywords: ["C4834"]
 
 Starting in the C++17 Standard, the `[[nodiscard]]` attribute specifies that a function's return value isn't intended to be discarded. If a caller discards the return value, the compiler generates warning C4834.
 
-To resolve this warning, consider why your code doesn't use the return value. Your use of the function may not match its intent. You can circumvent the warning by using a cast to **`void`**.
+To resolve this warning, consider why your code doesn't use the return value. Your use of the function may not match its intent. You can circumvent the warning by assigning the value to **`std::ignore`** or by using a cast to **`void`** (see [Warning C26457](../../code-quality/c26457.md)).
```

Comment:
It may be helpful to provide a small example here of assigning to std::ignore so that the reader doesn't have to google search how to do it.

Reply:
Never mind - you do below.

Reply:
As does the linked C26457 page :)